src/components/Todos/view.js

```diff
@@ -24,6 +26,8 @@ function renderMainSection(todosData) {
       checked: allCompleted
     }),
     ul('.todo-list', todosData.list
+      // Apply filter function according to
+      // route. 
```

Comment:
Correct, and this is not paraphrasing, **but** from the view we should make no assumption that it is "according to the route". The model is the only one who knows that information, the view should be agnostic. All the view knows is that there may be some filtering logic, and it is going to apply that.